RELEASENOTES-1.4.docu

```diff
@@ -526,4 +526,7 @@
   <build-id _6="6287" _7="7012"><add-note> Fixed a bug where overlapping
       registers would not be rejected if they were part of a bank within a group
       or subdevice.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note> Fixed a bug where executed callbacks
+      posted via immediate after statements would not trigger the device state
+      change notifier.</add-note></build-id>
```

Comment:
`state-change`?

Reply:
Not according to a releasenote in RELEASENOTES-1.2:
>  <build-id value="6122"><add-note> Added the <tt>--state-change-dml12</tt> flag, allowing the use of an incomplete version of state change notifiers in DML 1.2 devices <bug number="SIMICS-17952"/>.</add-note></build-id>